velox/exec/TableWriter.cpp

```diff
@@ -162,6 +162,9 @@ RowVectorPtr TableWriter::getOutput() {
 
   finished_ = true;
   updateWrittenBytes();
```

Comment:
Can we update inside updateWrittenBytes? so we grab the lock once? We can rename s/updateWrittenBytes/updateWrittenStats/